lambda/translmod.ml

```diff
@@ -1255,18 +1359,25 @@ let get_component = function
     None -> Lconst const_unit
   | Some id -> Lprim(Pgetglobal id, [], Loc_unknown)
 
+let () =
+  match Jkind.Sort.Const.for_module with
+  | Base Value -> ()
+  | _ -> Misc.fatal_error "Lambda.transl_package: expected modules to be values"
+    (* If this assumption is broken, [transl_package] should return a
+       module representation instead of a size *)
```

Comment:
This kind of sanity check is a bit atypical for the compiler, but I think it's OK here.  I considered suggesting we should remove it, but I do think our use of packages is thin enough that we could easily neglect to test this.

Please add a newline here, though!